spec/services/idv/proofing_components_spec.rb

```diff
@@ -38,6 +38,7 @@
       allow(FeatureManagement).to receive(:proofing_device_profiling_collecting_enabled?).
         and_return(true)
       idv_session.threatmetrix_review_status = 'pass'
+      idv_session.source_check_vendor = 'aamva'
```

Comment:
A/N: Is there a better way to test this or is the captured by the end_to_end_idv feature spec?

Reply:
I think the VerifyInfoController (or the VerifyInfoConcern? I'm not sure which) should have a test asserting that the value in the result object is propagated to the Idv Session

Reply:
You are right, will add that now.

Reply:
The closest we get to validating the result object is in the `end_to_end_idv` feature spec -- we have a method, `#validate_verify_info_submit`,  that validates the ProofingComponents structure, but not much else.

With respect to the controller specs, we primarily look at what value was logged in the event (with the exception of our threatmetrix status specs). 

For now, I've updated the VerifyInfoController spec to look at what was logged in the `state_id` stage and am open to expanding the coverage here.